src/controller/__init__.py

```diff
@@ -0,0 +1,60 @@
+MODULES = {
```

Comment:
Yes we can do that, I removed it because I didnt see the purpose of a json file, since we need python scripting in order to create a new module in any case.. 
But in fact its useful if we want to personalize the menu.